### ISSUE TITLE
blocked by whalespool. Please enable cookies

### ISSUE THREAD

**Issue by seevik2580** [Lolliedieb/lolMiner-releases#910]
hi,
i used lolminer for mining toncoin for 3 days with no problems at all, but today it starts to give me this response in terminal, but danila-miner works fine.

```
Pool https://server1.whalestonpool.com timeout or send defect message: "<!DOCTYPE html>
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en-US"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en-US"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en-US"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en-US"> <!--<![endif]-->
<head>
<title>Attention Required! | Cloudflare</title>
<meta charset="UTF-8" />
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1" />
<meta name="robots" content="noindex, nofollow" />
<meta name="viewport" content="width=device-width,initial-scale=1" />
<link rel="stylesheet" id="cf_styles-css" href="/cdn-cgi/styles/cf.errors.css" type="text/css" media="screen,projection" />
<!--[if lt IE 9]><link rel="stylesheet" id='cf_styles-ie-css' href="/cdn-cgi/styles/cf.errors.ie.css" type="text/css" media="screen,projection" /><![endif]-->
<style type="text/css">body{margin:0;padding:0}</style>


<!--[if gte IE 10]><!-->
<script>
  if (!navigator.cookieEnabled) {
    window.addEventListener('DOMContentLoaded', function () {
      var cookieEl = document.getElementById('cookie-alert');
      cookieEl.style.display = 'block';
    })
  }
</script>
<!--<![endif]-->


</head>
<body>
  <div id="cf-wrapper">
    <div class="cf-alert cf-alert-error cf-cookie-error" id="cookie-alert" data-translate="enable_cookies">Please enable cookies.</div>
    <div id="cf-error-details" class="cf-error-details-wrapper">
      <div class="cf-wrapper cf-header cf-error-overview">
        <h1 data-translate="block_headline">Sorry, you have been blocked</h1>
        <h2 class="cf-subheadline"><span data-translate="unable_to_access">You are unable to access</span> whalestonpool.com</h2>
      </div><!-- /.header -->

      <div class="cf-section cf-highlight">
        <div class="cf-wrapper">
          <div class="cf-screenshot-container cf-screenshot-full">

              <span class="cf-no-screenshot error"></span>

          </div>
        </div>
      </div><!-- /.captcha-container -->

      <div class="cf-section cf-wrapper">
        <div class="cf-columns two">
          <div class="cf-column">
            <h2 data-translate="blocked_why_headline">Why have I been blocked?</h2>

            <p data-translate="blocked_why_detail">This website is using a security service to protect itself from online attacks. The action you just performed triggered the security solution. There are several actions that could trigger this block including submitting a certain word or phrase, a SQL command or malformed data.</p>
          </div>

          <div class="cf-column">
            <h2 data-translate="blocked_resolve_headline">What can I do to resolve this?</h2>

            <p data-translate="blocked_resolve_detail">You can email the site owner to let them know you were blocked. Please include what you were doing when this page came up and the Cloudflare Ray ID found at the bottom of this page.</p>
          </div>
        </div>
      </div><!-- /.section -->

      <div class="cf-error-footer cf-wrapper w-240 lg:w-full py-10 sm:py-4 sm:px-8 mx-auto text-center sm:text-left border-solid border-0 border-t border-gray-300">
  <p class="text-13">
    <span class="cf-footer-item sm:block sm:mb-1">Cloudflare Ray ID: <strong class="font-semibold">6c7fad6c5afd701f</strong></span>
    <span class="cf-footer-separator sm:hidden">&bull;</span>
    <span class="cf-footer-item sm:block sm:mb-1"><span>Your IP</span>: ***************</span>
    <span class="cf-footer-separator sm:hidden">&bull;</span>
    <span class="cf-footer-item sm:block sm:mb-1"><span>Performance &amp; security by</span> <a rel="noopener noreferrer" href="https://www.cloudflare.com/5xx-error-landing" id="brand_link" target="_blank">Cloudflare</a></span>

  </p>
</div><!-- /.error-footer -->


    </div><!-- /#cf-error-details -->
  </div><!-- /#cf-wrapper -->

  <script type="text/javascript">
  window._cf_translation = {};


</script>

</body>
</html>
```

**Comment by jgonzis** [comment]
> hi, i used lolminer for mining toncoin for 3 days with no problems at all, but today it starts to give me this response in terminal, but danila-miner works fine.
> 
> ```
> Pool https://server1.whalestonpool.com timeout or send defect message: "<!DOCTYPE html>
> 
> 
> 
>  <html class="no-js" lang="en-US"> 
> <head>
> <title>Attention Required! | Cloudflare</title>
> <meta charset="UTF-8" />
> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
> <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1" />
> <meta name="robots" content="noindex, nofollow" />
> <meta name="viewport" content="width=device-width,initial-scale=1" />
> <link rel="stylesheet" id="cf_styles-css" href="/cdn-cgi/styles/cf.errors.css" type="text/css" media="screen,projection" />
> 
> <style type="text/css">body{margin:0;padding:0}</style>
> 
> 
> 
> <script>
>   if (!navigator.cookieEnabled) {
>     window.addEventListener('DOMContentLoaded', function () {
>       var cookieEl = document.getElementById('cookie-alert');
>       cookieEl.style.display = 'block';
>     })
>   }
> </script>
> 
> 
> 
> </head>
> <body>
>   <div id="cf-wrapper">
>     <div class="cf-alert cf-alert-error cf-cookie-error" id="cookie-alert" data-translate="enable_cookies">Please enable cookies.</div>
>     <div id="cf-error-details" class="cf-error-details-wrapper">
>       <div class="cf-wrapper cf-header cf-error-overview">
>         <h1 data-translate="block_headline">Sorry, you have been blocked</h1>
>         <h2 class="cf-subheadline"><span data-translate="unable_to_access">You are unable to access</span> whalestonpool.com</h2>
>       </div>
> 
>       <div class="cf-section cf-highlight">
>         <div class="cf-wrapper">
>           <div class="cf-screenshot-container cf-screenshot-full">
> 
>               <span class="cf-no-screenshot error"></span>
> 
>           </div>
>         </div>
>       </div>
> 
>       <div class="cf-section cf-wrapper">
>         <div class="cf-columns two">
>           <div class="cf-column">
>             <h2 data-translate="blocked_why_headline">Why have I been blocked?</h2>
> 
>             <p data-translate="blocked_why_detail">This website is using a security service to protect itself from online attacks. The action you just performed triggered the security solution. There are several actions that could trigger this block including submitting a certain word or phrase, a SQL command or malformed data.</p>
>           </div>
> 
>           <div class="cf-column">
>             <h2 data-translate="blocked_resolve_headline">What can I do to resolve this?</h2>
> 
>             <p data-translate="blocked_resolve_detail">You can email the site owner to let them know you were blocked. Please include what you were doing when this page came up and the Cloudflare Ray ID found at the bottom of this page.</p>
>           </div>
>         </div>
>       </div>
> 
>       <div class="cf-error-footer cf-wrapper w-240 lg:w-full py-10 sm:py-4 sm:px-8 mx-auto text-center sm:text-left border-solid border-0 border-t border-gray-300">
>   <p class="text-13">
>     <span class="cf-footer-item sm:block sm:mb-1">Cloudflare Ray ID: <strong class="font-semibold">6c7fad6c5afd701f</strong></span>
>     <span class="cf-footer-separator sm:hidden">&bull;</span>
>     <span class="cf-footer-item sm:block sm:mb-1"><span>Your IP</span>: ***************</span>
>     <span class="cf-footer-separator sm:hidden">&bull;</span>
>     <span class="cf-footer-item sm:block sm:mb-1"><span>Performance &amp; security by</span> <a rel="noopener noreferrer" href="https://www.cloudflare.com/5xx-error-landing" id="brand_link" target="_blank">Cloudflare</a></span>
> 
>   </p>
> </div>
> 
> 
>     </div>
>   </div>
> 
>   <script type="text/javascript">
>   window._cf_translation = {};
> 
> 
> </script>
> 
> </body>
> </html>
> ```




> hi, i used lolminer for mining toncoin for 3 days with no problems at all, but today it starts to give me this response in terminal, but danila-miner works fine.
> 
> ```
> Pool https://server1.whalestonpool.com timeout or send defect message: "<!DOCTYPE html>
> 
> 
> 
>  <html class="no-js" lang="en-US"> 
> <head>
> <title>Attention Required! | Cloudflare</title>
> <meta charset="UTF-8" />
> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
> <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1" />
> <meta name="robots" content="noindex, nofollow" />
> <meta name="viewport" content="width=device-width,initial-scale=1" />
> <link rel="stylesheet" id="cf_styles-css" href="/cdn-cgi/styles/cf.errors.css" type="text/css" media="screen,projection" />
> 
> <style type="text/css">body{margin:0;padding:0}</style>
> 
> 
> 
> <script>
>   if (!navigator.cookieEnabled) {
>     window.addEventListener('DOMContentLoaded', function () {
>       var cookieEl = document.getElementById('cookie-alert');
>       cookieEl.style.display = 'block';
>     })
>   }
> </script>
> 
> 
> 
> </head>
> <body>
>   <div id="cf-wrapper">
>     <div class="cf-alert cf-alert-error cf-cookie-error" id="cookie-alert" data-translate="enable_cookies">Please enable cookies.</div>
>     <div id="cf-error-details" class="cf-error-details-wrapper">
>       <div class="cf-wrapper cf-header cf-error-overview">
>         <h1 data-translate="block_headline">Sorry, you have been blocked</h1>
>         <h2 class="cf-subheadline"><span data-translate="unable_to_access">You are unable to access</span> whalestonpool.com</h2>
>       </div>
> 
>       <div class="cf-section cf-highlight">
>         <div class="cf-wrapper">
>           <div class="cf-screenshot-container cf-screenshot-full">
> 
>               <span class="cf-no-screenshot error"></span>
> 
>           </div>
>         </div>
>       </div>
> 
>       <div class="cf-section cf-wrapper">
>         <div class="cf-columns two">
>           <div class="cf-column">
>             <h2 data-translate="blocked_why_headline">Why have I been blocked?</h2>
> 
>             <p data-translate="blocked_why_detail">This website is using a security service to protect itself from online attacks. The action you just performed triggered the security solution. There are several actions that could trigger this block including submitting a certain word or phrase, a SQL command or malformed data.</p>
>           </div>
> 
>           <div class="cf-column">
>             <h2 data-translate="blocked_resolve_headline">What can I do to resolve this?</h2>
> 
>             <p data-translate="blocked_resolve_detail">You can email the site owner to let them know you were blocked. Please include what you were doing when this page came up and the Cloudflare Ray ID found at the bottom of this page.</p>
>           </div>
>         </div>
>       </div>
> 
>       <div class="cf-error-footer cf-wrapper w-240 lg:w-full py-10 sm:py-4 sm:px-8 mx-auto text-center sm:text-left border-solid border-0 border-t border-gray-300">
>   <p class="text-13">
>     <span class="cf-footer-item sm:block sm:mb-1">Cloudflare Ray ID: <strong class="font-semibold">6c7fad6c5afd701f</strong></span>
>     <span class="cf-footer-separator sm:hidden">&bull;</span>
>     <span class="cf-footer-item sm:block sm:mb-1"><span>Your IP</span>: ***************</span>
>     <span class="cf-footer-separator sm:hidden">&bull;</span>
>     <span class="cf-footer-item sm:block sm:mb-1"><span>Performance &amp; security by</span> <a rel="noopener noreferrer" href="https://www.cloudflare.com/5xx-error-landing" id="brand_link" target="_blank">Cloudflare</a></span>
> 
>   </p>
> </div>
> 
> 
>     </div>
>   </div>
> 
>   <script type="text/javascript">
>   window._cf_translation = {};
> 
> 
> </script>
> 
> </body>
> </html>
> ```

LolMiner TON Whales Servers:
✔️Server: pool.services.tonwhales.com
✔️Server: ton-pool-server-p3agi.ondigitalocean.app
✖️server1.whalestonpool.com

**Comment by jgonzis** [comment]
Sure will be a future release to solve it

**Comment by EdSkjavez** [comment]
I have exactly the same problem in two of my rigs, back to danila miner witch is a problem because i cant exclude cards for rendering

**Comment by mr-scrpt** [comment]
I have the same error on windows. It works on hive for now. What to do? Tonwhales pool

**Comment by yegresoknertom** [comment]
Faced exactly the same issue in HiveOS on tonwhales pool. After several reboots, the miner has started to work normally.

**Comment by jgonzis** [comment]
> LolMiner TON Whales Servers:
> ✔️Server: pool.services.tonwhales.com
> ✔️Server: ton-pool-server-p3agi.ondigitalocean.app
> ✖️server1.whalestonpool.com

use Pools with the tick !! the server1.whalestonpool.com is not in the best situation

**Comment by seevik2580** [comment]
thanks for help, this works for me :)

**Comment by seevik2580** [comment]
happened to me again on https://pool.services.tonwhales.com

**Comment by jgonzis** [comment]
> happened to me again on https://pool.services.tonwhales.com

Could you try?
http://ton-pool-server-p3agi.ondigitalocean.app

**Comment by seevik2580** [comment]
http://ton-pool-server-p3agi.ondigitalocean.app doesnt work at all for me, in browser i see `Pool is working. Have a nice day :)` but lolminer crashes after message Connecting to pool...

**Comment by mavaro** [comment]
The same here. lolminer is not working with any of the three servers...

**Comment by mavaro** [comment]
Error:
`Pool https://ton-pool-server-p3agi.ondigitalocean.app timeout or send defect message: "upstream connect error or disconnect/reset before headers. reset reason: connection failure"`

**Comment by seevik2580** [comment]
but danila miner works fine with urls, why not lolminer. is lolminer ddosing those servers ? :D

**Comment by seevik2580** [comment]
> > The same here. lolminer is not working with any of the three servers...
> 
> It seems another attack DDOS... Could you restart the please?

still not working

**Comment by mavaro** [comment]
Restarted with no luck. lolminer is not working with any of the three servers... I don't use danila-miner, so I don't know it's working or not...

**Comment by szabokacsa** [comment]
> http://ton-pool-server-p3agi.ondigitalocean.app doesnt work at all for me, in browser i see `Pool is working. Have a nice day :)` but lolminer crashes after message Connecting to pool...

It crashes because it is "https" not "http"

I also tried https://ton-pool-server-p3agi.ondigitalocean.app and it is working, some shares are already found but the lolminer does not show any accepted share.

**Comment by seevik2580** [comment]
> It crashes because it is "https" not "http"

yes i tried both, http crash, https gives me `Pool https://ton-pool-server-p3agi.ondigitalocean.app timeout or send defect message: "upstream connect error or disconnect/reset before headers. reset reason: connection termination"`

**Comment by mavaro** [comment]
> yes i tried both, http crash, https gives me `Pool https://ton-pool-server-p3agi.ondigitalocean.app timeout or send defect message: "upstream connect error or disconnect/reset before headers. reset reason: connection termination"`

The same here. The other two servers, not working, with cloudfare blocking me.

**Comment by seevik2580** [comment]
my mistake, danila miner works only with 
https://pool.services.tonwhales.com and https://server1.whalestonpool.com, last url https://ton-pool-server-p3agi.ondigitalocean.app doesnt work right now even with danila miner. just fyi

**Comment by jgonzis** [comment]
Another time a DDOS... let's wait admins to resolve that problem...

**Comment by mavaro** [comment]
> Another time a DDOS... let's wait admins to resolve that problem...

Thank you for the info.

**Comment by jgonzis** [comment]
> > Another time a DDOS... let's wait admins to resolve that problem...
> 
> Thank you for the info.

Sure tomorrow will be a solution to mine in ton Whales and not change to ton-pool

**Comment by seevik2580** [comment]
> Another time a DDOS... let's wait admins to resolve that problem...

https://github.com/Lolliedieb/lolMiner-releases/issues/914
`Here is a quote from the developer:
⚠️ Lolminer is blocked by our anti-DDOS system because it sent x1000+ request comparing to Danila miner.`

 looks like lolminer is the one who make those requests :(

**Comment by jgonzis** [comment]
> > Another time a DDOS... let's wait admins to resolve that problem...
> 
> #914 `Here is a quote from the developer: ⚠️ Lolminer is blocked by our anti-DDOS system because it sent x1000+ request comparing to Danila miner.`
> 
> looks like lolminer is the one who make those requests :(

All do it, the attack is easy just do a https request... lot of time. Servers seem up another time....

**Comment by jgonzis** [comment]
SOLVED with lolMiner 1.40, update and enjoy mining!!
